packages/dbos-cloud/cli.ts

```diff
@@ -341,45 +333,49 @@ dashboardCommands
 /* ORGANIZATIONS COMMANDS */
 ////////////////////////////
 
-const orgCommands = program
-  .command('organization')
-  .alias('organizations')
-  .alias('org')
-  .description('Manage dbos organizations')
+const orgCommands = program.command("organization").alias("organizations").alias("org").description("Manage dbos organizations");
```

Comment:
Sorry for the noisy changes, it's the `.editorconfig` being finally applied on this package.